lib/ecto.ex

```diff
@@ -397,6 +397,8 @@ defmodule Ecto do
 
   """
 
+  def version, do: unquote(Mix.Project.config[:version])
```

Comment:
Borrowed that from hex actually https://github.com/hexpm/hex/blob/master/lib/hex.ex#L34